<launch>

	<!-- Participant ID -->
	<arg name="ID" default="0"/>

	<!-- Task -->
	<arg name="task" default="None"/> 

	<!-- Method type ('A'- impedance, 'B'- pHRI_learning, 'C'- demonstration_learning, 'D'- discrete_demonstration_learning, 'E'- expert) -->
	<arg name="method_type" default="A"/>

	<!-- Record data mode -->
	<arg name="record" default="F"/>

	<!-- Replay data mode -->
	<arg name="replay" default="F"/>

	<!-- Simulate data mode -->
	<arg name="simulate" default="F"/>	

	<!-- Feature learning method (ALL, MAX, BETA)-->
	<arg name="feat_method" default="ALL"/>

	<!-- List of string features that need correcting -->
	<arg name="feat_list" default="table"/>

	<!-- velocity-based with trajopt planning PID -->
	<node name="pid_trajopt" pkg="beta_adaptive_pHRI" type="pid_trajopt.py"
    args="$(arg ID) $(arg task) $(arg method_type) $(arg record) $(arg replay)
    $(arg simulate) $(arg feat_method) $(arg feat_list)" output="screen"/>

</launch>
